tests/gis_tests/gdal_tests/test_driver.py

```diff
@@ -48,3 +49,13 @@ def test03_aliases(self):
         for alias, full_name in aliases.items():
             dr = Driver(alias)
             self.assertEqual(full_name, str(dr))
+
+    @mock.patch('django.contrib.gis.gdal.driver.vcapi.get_driver_count')
+    @mock.patch('django.contrib.gis.gdal.driver.rcapi.get_driver_count')
+    @mock.patch('django.contrib.gis.gdal.driver.vcapi.register_all')
+    def test04_registered(self, vreg, rcount, vcount):
+        "Testing driver registration"
```

Comment:
State the expected behavior, e.g. "Prototypes are registered only if their respective driver counts aren't zero."